relay/pkg/node/node.go

```diff
@@ -31,39 +32,39 @@ func Initialize(
 		stats: newStats(),
 	}
 
-	go node.runForwarderControlLoop(ctx, btcChain, hostChain)
+	go node.startRelayController(ctx, btcChain, hostChain)
 
 	return node
 }
 
-// runForwarderControlLoop runs a block forwarder control loop which is
-// responsible for starting the forwarder and acting upon errors by restarting
-// the forwarder instance. The lifecycle of the control loop itself can
+// startRelayController starts a headers relay controller which is
+// responsible for starting the relay and acting upon errors by restarting
+// the relay instance. The lifecycle of the controller itself can
 // be managed using the passed context.
-func (n *Node) runForwarderControlLoop(
+func (n *Node) startRelayController(
```

Comment:
I like the `controlLoop` version. It is clear what is the purpose and how it's supposed to work (keep iterating and starting on error).

Reply:
:tada: